Build/Build.cs

```diff
@@ -70,9 +68,18 @@ class Build : NukeBuild
     AbsolutePath TestResultsDirectory => RootDirectory / "TestResults";
 
     string SemVer;
-    string YarnCli => ToolPathResolver.GetPackageExecutable("Yarn.MSBuild", "yarn.js", "1.22.19");
+
+    Target Informations => _ => _
+        .OnlyWhenDynamic(() => !IsLocalBuild)
+        .Executes(() =>
+        {
+            Information(BranchSpec);
+            Information(BuildNumber);
+            Information(PullRequestBase);
+        });
 
     Target Clean => _ => _
+        .Triggers(Informations)
```

Comment:
Does `Informations` run independently on whether `SpellCheck` or `Push` is run?

Reply:
No I don't think so.. is this a problem?

Reply:
Ah.. got it.. will change that

Reply:
What about leaving this target, but do not call it at all? Just to invoke it via cli switch `build.sh --target informations`?